fdbclient/include/fdbclient/StorageServerInterface.h

```diff
@@ -1293,6 +1297,26 @@ struct GetStorageCheckSumRequest {
 	}
 };
 
+struct BulkDumpRequest {
+	constexpr static FileIdentifier file_identifier = 3828145;
```

Comment:
What are these for?

Reply:
This is for serializing and deserializing the data structure. See ObjectSerializer.h for details.

Reply:
Thanks